framework/test-src/play/testlog4j.properties

```diff
@@ -1,9 +1,16 @@
-log4j.rootLogger=DEBUG, PropertiesConsoleAppender
-
-log4j.appender.PropertiesConsoleAppender=org.apache.log4j.ConsoleAppender
-log4j.appender.PropertiesConsoleAppender.layout=org.apache.log4j.PatternLayout
-log4j.appender.PropertiesConsoleAppender.layout.ConversionPattern=%d{ISO8601} %-5p [%t] %c: %m%n
-
-
-
-log4j.logger.logtest.properties=ERROR
+# appender Console
+appender.console.type = Console

```

Comment:
Could you replace CRLF by LF ?

Reply:
done in https://github.com/playframework/play1/pull/1380

Reply:
i changed in a64ab7c417a67c074e201f842097722791cff88f
and add solution --> when configuration not working use autoconfiguration